src/backupNg/backupNg.spec.js

```diff
@@ -232,4 +232,64 @@ describe("backupNg", () => {
       expect(vmTask.data.id).toBe(vmIdWithoutDisks);
     });
   });
+
+  test("execute two times a rolling snapshot with 2 as retention & revert to an old state", async () => {
+    jest.setTimeout(7e4);
+    const SR = await xo.getSrId();
+    const vmId = await xo.createTempVm({
+      name_label: "XO Test Temp",
+      name_description: "Creating a temporary vm",
+      template: config.xoTestTemplateId,
+      VDIs: [
+        {
+          device: "0",
+          size: 1,
+          SR,
+          type: "user",
+        },
+      ],
+    });
+
+    const scheduleTempId = randomId();
+    const { id: jobId } = await xo.createTempBackupNgJob({
+      ...defaultBackupNg,
+      vms: {
+        id: vmId,
+      },
+      schedules: {
+        [scheduleTempId]: {
+          name: "scheduleTest",
```

Comment:
You can create a global constant for this default schedule.

Reply:
Can I do this change in another PR to modify in the same time all previous test using `scheduleTest` ?

Reply:
The change will be related to this PR and it will not add a lot of line, so i think better do it in this PR.

Reply:
Ok :+1: